pkg/httpd/tty.go

```diff
@@ -372,7 +372,7 @@ func (h *tty) getShareUserInfo(query GetUserParams) {
 		logger.Errorf("Ws[%s] get share User info without permissions", h.ws.Uuid)
 		return
 	}
-	shareUserResp, err := h.ws.apiClient.GetShareUserInfo(query.Query)
+	shareUserResp, err := h.ws.apiClient.GetSuggestionUsers(query.Query)
 	if err != nil {
 		logger.Error(err)
 		return
```

Comment:
There is no need for any changes to this code snippet. It seems that the provided code corresponds to a Go function called "getShareUserInfo". This function takes a `GetUserParams` object representing user parameters and uses an API client (`ws.apiClient`) belonging to an instance of type TTY (type hint). The purpose of the function appears not to be defined in this context but it does use some standard Go methods.

Since there are no errors in this section, I will assume that these functions are correct implementations based on the information available at hand. Please verify if you have more details about the functionality of this function.